docs/upgrade/upgrade-opensource/upgrade-guide-from-5.2-to-5.3/upgrade-guide-from-5.2-to-5.3-generic.rst

```diff
@@ -0,0 +1,432 @@
+
+        * :ref:`Upgrading ScyllaDB without updating any external packages <upgrade-image-upgrade-guide-regular-procedure>`.
+
+        .. _upgrade-image-recommended-procedure:
+
+        **To upgrade ScyllaDB and update 3rd party and OS packages (RECOMMENDED):**
+
+        Choosing this upgrade procedure allows you to upgrade your ScyllaDB version and update the 3rd party and OS packages using one command.
+
+        #. Update the |SCYLLA_DEB_NEW_REPO| to |NEW_VERSION|.
+
+        #. Load the new repo:
+
+            .. code:: sh
+
+               sudo apt-get update
+
+
+        #. Run the following command to update the manifest file:
+
+            .. code:: sh
+
+               cat scylla-packages-<version>-<arch>.txt | sudo xargs -n1 apt-get install -y
+
+            Where:
+
+              * ``<version>`` - The ScyllaDB version to which you are upgrading ( |NEW_VERSION| ).
+              * ``<arch>`` - Architecture type: ``x86_64`` or ``aarch64``.
+
+            The file is included in the ScyllaDB packages downloaded in the previous step. The file location is ``http://downloads.scylladb.com/downloads/scylla/aws/manifest/scylla-packages-<version>-<arch>.txt``
+
+            Example:
+
+                .. code:: sh
+
+                   cat scylla-packages-5.2.0-x86_64.txt | sudo xargs -n1 apt-get install -y
+
+                .. note::
+
+                   Alternatively, you can update the manifest file with the following command:
+
+                   ``sudo apt-get install $(awk '{print $1'} scylla-packages-<version>-<arch>.txt) -y``
+
+        .. _upgrade-image-upgrade-guide-regular-procedure:
+
+        **To upgrade ScyllaDB:**
+
+        #. Update the |SCYLLA_DEB_NEW_REPO| to |NEW_VERSION|.
+
+        #. Install the new ScyllaDB version:
+
+            .. code-block:: console
+
+               sudo apt-get clean all
+               sudo apt-get update
+               sudo apt-get dist-upgrade scylla
+
+
+        Answer ‘y’ to the first two questions.
+
+.. _upgrade-5.3-enable-raft:
+
+(Optional) Enable consistent cluster management in the node's configuration file
+--------------------------------------------------------------------------------
+
+.. note::
+
+   Skip this section if you enabled Raft-based consistent cluster management in version 5.2.
+
+Optionally, you can enable Raft-based consistent cluster management on your cluster with the ``consistent_cluster_management``
+option. Setting the option to ``true`` on every node enables the Raft consensus algorithm for your cluster. Raft will be used
+to consistently manage cluster-wide metadata as soon as you finish upgrading every node to the new version. See 
+:doc:`Raft in ScyllaDB </architecture/raft/>` to learn more.
+
+In ScyllaDB 5.2 and 5.3, Raft-based consistent cluster management is disabled by default. If you didn't enable the feature
```

Comment:
I thought for new deployments it's enabled by default?

Reply:
What Yaniv meant is that in 5.2/5.3, it's enabled by default in **new deployments** i.e. when you start a completely new cluster; it's not yet enabled by default during upgrade (this is what the decision that needs to be made is about).

Indeed, we should probably say something like (but this can be fixed in a follow-up - I already queued the patch)
"When you upgrade to ScyllaDB 5.2 and 5.3, Raft-based consistent cluster management is disabled by default"

- to distinguish from the situation of starting a new cluster (where it will be enabled by default because of provided conf/scylla.yaml which enables it)

Reply:
Thanks, I get it now. You're both right - I'll clarify the information in the follow-up PR.